Client/Frontend/Popover/PopoverController.swift

```diff
@@ -459,41 +527,27 @@ extension PopoverController: UIGestureRecognizerDelegate {
     }
     
     func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
-        guard let pan = gestureRecognizer as? UIPanGestureRecognizer else {
-            return false
-        }
-        
+
         // Don't allow for pan gesture while table view is in editing mode.
         // This bug only occurs for bookmarks at nested levels, see issue #687.
         if let tableView = otherGestureRecognizer.view as? UITableView, tableView.isEditing {
             gestureRecognizer.cancel()
```

Comment:
Do you think this is still required? I think we can probably just make the entire function return false

Reply:
right, it actually seems to return false by default so can remove whole function